sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/GoneAndRetryWithRetryPolicy.java

```diff
@@ -23,8 +24,8 @@
 import java.time.Duration;
 
 public class GoneAndRetryWithRetryPolicy extends RetryPolicyWithDiagnostics {
-
     private final static Logger logger = LoggerFactory.getLogger(GoneAndRetryWithRetryPolicy.class);
+    private final static int LIMITED_STACK_TRACE_FAILURE_DEPTH = 3;
```

Comment:
It would be better if we can have this as a JVM parameter as well defined in `Configs.java`

So that if we want the customer to increase the depth size, we don't have to change the code, release and ask them to upgrade. 

With JVM params, they can just update the param value and they can re-start their application.

Reply:
moved to Configs.java